### PR TITLE
fix: navigation, pagination, scrollbar props #4181

### PR DESCRIPTION
**Pull request by mya-ake**
fix: #4181 

In the case of `{}`, `needsNavigation`, `needsPagination`, and `needsScrollbar` will be true, so if each props is false, the initial value is given so that element is null.

### CODE REVIEW

File: src/react/get-params.js [hunk 1]
```diff
@@ -33,7 +33,17 @@ function getParams(obj = {}) {
     }
   });
   ['navigation', 'pagination', 'scrollbar'].forEach((key) => {
-    if (params[key] === true || params[key] === false) params[key] = {};
```
Comment (vltansky) [inline src/react/get-params.js:36]: Why do you need to specify default value?
Wouldnt it be enough to change it to:
```js
if (params[key] === true) params[key] = {};
```

**Comment by vltansky** [comment]
You can contribute to vue & svelte versions with the same change.

Take a look at `src/svelte/get-params.js` and `src/vue/get-params.js`

File: src/react/get-params.js (hunk 1 above)
Comment (mya-ake) [reply src/react/get-params.js:36]: I wondered if I needed to convert it to an object, so I set the default value.
That change seems to be enough, so I will change it.

**Comment by mya-ake** [comment]
I also updated svelte and vue.

**Comment by nolimits4web** [comment]
Merged, thanks! Checked and everything seems fine

**Pull request merged**